schema_based_on_diagram.sql

```diff
@@ -65,6 +65,18 @@ CREATE INDEX ON invoice_items (treatment_id);
 
 CREATE INDEX ON invoices (medical_history_id);
 
+
 CREATE INDEX ON medical_histories (patient_id);
 
-CREATE INDEX ON medical_histories (treatment_id);
\ No newline at end of file
+CREATE INDEX ON medical_histories (treatment_id);
+
+/* Commands to alter the medical_histories table foreign key constraints with patient_id and treatment_id columns */
+
+ALTER TABLE medical_histories DROP CONSTRAINT fk_patients;
```

Comment:
- [ ] Great work on removing the foreign key of treatments from the medical_histories table 👏 but I guess you also removed the foreign key for patients which is not asked by the previous reviewer 🤔 Kindly put the foreign key for patients back in the medical_histories table 😊 because if you don't do this then it would broke your medical_histories and patients table one-to-many relationship 😨